### ISSUE TITLE
Adding "Preview and Save" Feature to Photo Affix

### ISSUE THREAD

**Issue by adewararilwan** [afollestad/photo-affix#28]
I will love to propose a new feature that is needed to be implemented.
- Preview  and Save
### </center>My Reason for proposing the "Preview  and Save" feature</center>
When operating the Photo Affix application, I chose the pictures I wanted to merge together and I click on the Affix button then it saves immediately. I went on checking affixed photos on my Gallery and I notice I was not satisfied with the way I Affixed it.

#### - Preview and Save 
 It will be a very good idea if I am able to view my affixed photo before being saved to my Internal Storage/SD card to avoid the stress of repeating affixing photos if not contented with what I have.

## <center>MOCKUP EXAMPLE</center>
![Mockup.jpg](https://cdn.steemitimages.com/DQmXDqipzm9YeU4Rxt9yy2HGsMVSoo8U5iUC2PFq47NGQdU/Mockup.jpg)

## BENEFIT
*Preview and save*  
This will be a very important feature that will improve the application. 
If the Preview and Save feature is implemented to the Photo Affix application, it will allow the user to make a necessary inspection to the image merged before being saved.  It will save users from the stress of starting affixation all over again if they are not satisfied with how images are been affixed.
As of recent,  users that are using the application can only pick the image they want to merge and get it to affix, it lacks inspection before saving. And the situation where they are not satisfied with what they have, they will need to go on to the gallery and delete the unwanted image and frustratingly they have to start affixation from the beginning. 
Adding the Preview and save feature will be very helpful.

##### In conclusion,  
If the proposed feature is implemented, it will really improve the application and most especially increase the adoption of the application.

## MY GITHUB PROFILE:
https://github.com/adewararilwan
